### DISCUSSION TITLE
What are all of the MOTD placeholders?

### DISCUSSION THREAD

**Discussion by jmdiegel707** [Pryaxis/TShock#2456] [Q&A] [answered]
I know of the placeholders `%onlineplayers` and stuff, but are there more?

**Comment by moisterrific** [comment, marked as answer]
There is `%map%` for the map name, or server name if you have that enabled in config.
`%players%` lists the name of currently online players and joins their names like "`player1`, `player2`, `player3`, etc.".
`%specifier%` matches whatever command specifier you have set in config.
`%onlineplayers%` shows the number of players online. 
`%serverslots%` shows the server's player cap set via config. 

You can find it in the source code [here](https://github.com/Pryaxis/TShock/blob/03c20a98048ee7493d58140c0b8458a3d4399484/TShockAPI/TSPlayer.cs#L1528).

Also feel free to suggest other "placeholder" stuff, if they're doable and make sense to add they will most likely be implemented in the future.
